e2e/README.md

```diff
@@ -60,15 +60,26 @@ following environment variables can be exported to customize kubernetes deployme
 - creating kubernetes  cluster
 
 ```console
-$./minikube.sh up
+$ ./scripts/minikube.sh up
```

Comment:
please mention that script has to be invoked from ceph-csi root directory

Reply:
done now :-)